src/components/TooltipGlobal/TooltipGlobal.integration.test.js

```diff
@@ -0,0 +1,63 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { MockJetsContextProvider } from '../../__mocks__/MockJetsContext';
+
+import { JetsTooltipGlobal } from './index';
+import { activeTooltipData, paramsData } from './__fixtures__';
+
+const setup = ({ data = {}, config = {}, params = {}, events = {} } = {}) => ({
+  user: userEvent.setup(),
+  ...render(
+    <MockJetsContextProvider config={config} params={{ ...paramsData(params) }} events={events}>
+      <JetsTooltipGlobal data={{ ...activeTooltipData(data) }} />
+    </MockJetsContextProvider>
+  ),
+});
+
+describe('JetsTooltipGlobal', () => {
+  describe('when a button in the tooltip is pressed', () => {
+    it('should fire onSeatSelect when "Select" is pressed', async () => {
+      const onSeatSelect = jest.fn();
+      const { user } = setup({ events: { onSeatSelect } });
+
+      await user.click(screen.getByText(/Select/));
+      expect(onSeatSelect).toHaveBeenCalledTimes(1);
```

Comment:
Could we assert that the onSeatSelect event is called with the expected data please?